ephios/event_management/views.py

```diff
@@ -134,6 +141,82 @@ def get_queryset(self):
         )
 
 
+class EventCopyView(CustomPermissionRequiredMixin, SingleObjectMixin, FormView):
+    permission_required = "event_management.add_event"
+    model = Event
+    template_name = "event_management/event_copy.html"
+    form_class = EventDuplicationForm
+
+    def setup(self, request, *args, **kwargs):
+        super().setup(request, *args, **kwargs)
+        self.object = self.get_object()
+
+    def form_valid(self, form):
+        occurences = form.cleaned_data["recurrence"].between(
+            datetime.now() - timedelta(days=1),
+            datetime.now() + timedelta(days=365),
```

Comment:
Does this mean I can't create events in 2022? Seems unwanted to me.

Reply:
Yes.
It was meant as a precaution to prevent users from creating events that recurr forever. As we dropped the event series, it will be very hard to delete all of these afterwards. Do you have a good alternative for that?

Reply:
Well I thought we'd add a feature to bulk delete events by filtering for type+date_from+date_to. That would enable people to undo that.

Reply:
Hm, kinda complicated. You'd need to select a start date but no end date in the filter and make sure that there are no legitimate events in between that you dont't want to delete

Reply:
Well we need *some* kind of bulk delete. Until then, I'd suggest to increase the limit to two years, so you can at least create events for the whole upcoming year.

Reply:
Even better, enforce that a recurrence end is set in the front end.

Reply:
limit is now two years, enforcing an end date is moved to #195